pom.xml

```diff
@@ -1505,22 +1515,22 @@ xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xs
       <dependency>
         <groupId>org.javassist</groupId>
         <artifactId>javassist</artifactId>
-        <version>3.21.0-GA</version>
+        <version>${javassist.version}</version>
       </dependency>
       <dependency>
         <groupId>org.glassfish.hk2.external</groupId>
         <artifactId>jakarta.inject</artifactId>
-        <version>2.6.1</version>
+        <version>${jakarta.inject.version</version>
```

Comment:
Thank you for the contribution, @nishitpatira! Could you please enclose `${jakarta.inject.version` with a closing curly brace? Thanks.

Reply:
Hey, sorry, my license for IntelliJ had expired and my local build was failing, so, thought of running it on Actions for getting a better report. Fixed it.